mParticleCore.brs

```diff
@@ -1320,6 +1340,30 @@ function mParticleStart(options as object, messagePort as object)
             return message
         end function,
 
+        UserAttributeChange: function(userAttributeKey as string, newValue as object, oldValue as object, deleted as boolean, isNewAttribute as boolean) as object
+            message = m.Message(mParticleConstants().MESSAGE_TYPE.USER_ATTRIBUTE_CHANGE)
+            message.n = userAttributeKey
+            message.nv = newValue
+            message.ov = oldValue
+            message.d = deleted
+            message.na = isNewAttribute
+            currentMpid = mparticle()._internal.storage.getCurrentMpid()
+            message.ua = mparticle()._internal.storage.getUserAttributes(currentMpid)
+
+            return message
+        end function,
+
+        UserIdentityChange: function(newIdentity as object, oldIdentity as object) as object
+            message = m.Message(mParticleConstants().MESSAGE_TYPE.USER_IDENTITY_CHANGE)
+            message.ni = newIdentity
+            message.oi = oldIdentity
+            currentMpid = mparticle()._internal.storage.getCurrentMpid()
+            identities = mparticle()._internal.storage.getUserIdentities(currentMpid)
+            message.ui = identities
```

Comment:
same here - I don't think this is part of the json contract

Reply:
Yeah I don't see these in the Live stream but, I was matching what Android sent and thought it just might not be shown on Livestream. https://github.com/mParticle/mparticle-android-sdk/blob/43889c3da08c8daf8bf024f7e85ab3796203ce2c/android-core/src/main/java/com/mparticle/internal/MessageManager.java#L700